api-client/lib/api/extensions/get_into_teaching_api_client/api_client.rb

```diff
@@ -15,11 +15,20 @@ module ApiClient
 
       def faraday
         Faraday.new do |f|
+          if config.circuit_breaker&.dig(:enabled)
```

Comment:
```suggestion
          if config.circuit_breaker[:enabled]
```

I think this can be simplified a little, `#dig` for a single level is perhaps overkill?

Reply:
I suggested this as I don't _think_ you can do safe navigation on a hash directly, so `circuit_breaker&[:enabled]` and it felt neater than `circuit_breaker != nil && ...` there's maybe a better way I haven't thought of though 🤷‍♂️